arbeitszeit_web/presenters/register_member_presenter.py

```diff
@@ -0,0 +1,34 @@
+from dataclasses import dataclass
+
+from arbeitszeit.use_cases.register_member import RegisterMemberUseCase
+from arbeitszeit_web.register_member import RegisterForm
+from arbeitszeit_web.session import Session
+from arbeitszeit_web.translator import Translator
+
+
+@dataclass
+class RegisterMemberViewModel:
+    is_success_view: bool
+
+
+@dataclass
+class RegisterMemberPresenter:
+    session: Session
+    translator: Translator
+
+    def present_member_registration(
+        self, response: RegisterMemberUseCase.Response, form: RegisterForm
+    ) -> RegisterMemberViewModel:
+        if response.is_rejected:
+            if (
+                response.rejection_reason
+                == RegisterMemberUseCase.Response.RejectionReason.member_already_exists
+            ):
+                form.add_email_error(
+                    self.translator.gettext("This email address is already registered.")
+                )
+            return RegisterMemberViewModel(is_success_view=False)
+        else:
+            email = form.get_email_string()
+            self.session.login_member(email)
```

Comment:
i guess conceptually a better place for the log-in would be in the use case, right? is it a pragmatic decision to deviate from the rule here?

Reply:
I'd argue that that "logging the user in" is something that should be handled in the `arbeitszeit_web` package since it is specific to the web platform. If the application was a desktop program then the "logging in" wouldn't make sense necessarily.

Reply:
Ok!